js/classes/entities/foreground/dialog/conversation.js

```diff
@@ -35,14 +35,17 @@ class Conversation {
                     // player wants to move onto next bubble.
 
                     currentBubble.removeFromWorld = true;
+                    if (currentBubble.callback) {
+                        currentBubble.callback();
```

Comment:
What does callback and callback() do? It's a bit new to me. Is this built  in the JS?

Reply:
No. Look at the dialog bubble class - callback is an optional parameter, which says "After this dialog bubble is removed from world, do callback()" It's an optional parameter, hence if (currentBubble.callback)